netlify.toml

```diff
@@ -34,4 +34,10 @@
 from = "/tags/*"
 to = "/tag/:splat"
 status = 301
-force = false
\ No newline at end of file
+force = false
+
+[[redirects]]
```

Comment:
This config should be moved to the netlify redirects file 👍 it's just that the toml will be deleted soon when we reconfigure the netlify build